### DISCUSSION TITLE
Why is XML namespaces not allowed?

### DISCUSSION THREAD

**Discussion by ch99q** [swc-project/swc#6863] [General]
Hello, SWC community,

I noticed that XML namespaces are not currently allowed in the project. This means that projects that depend on SWC will throw an error on this, causing inconvenience and confusion. However, I understand that this is causing problems for frameworks that rely on namespaces in their attributes, which is supported by TypeScript 4.2 ([#11833](https://github.com/microsoft/TypeScript/issues/11833)) ([#37421](https://github.com/microsoft/TypeScript/pull/37421)).

I would like to know more about why this decision was made and if there are any plans to address this issue. It would be great to hear any thoughts or explanations on this matter.

**Comment by kdy1** [comment]
I just followed behavior of babel. Removing the restriction is fine, but I don't think it's a high priority.
